runner/run.cpp

```diff
@@ -212,6 +212,7 @@ float* forward(Transformer* transformer, int token, int pos) {
                              .to(torch::dtype(torch::kFloat32))
                              .to(torch::kCPU);
   auto logits = result[0].data_ptr();
+  memcpy(s->logits, logits, p->vocab_size * sizeof(float));
```

Comment:
This is for AOTI right? This is needed because the rest of the runner assumes float?

Reply:
it is simply moved up from below.